packages/app-plugin-security-cognito/src/identityProvider/index.tsx

```diff
@@ -47,12 +59,19 @@ export default (): PluginCollection => [
     {
         type: "security-user-form",
         render({ Bind, data }) {
+            const passwordValidators = [];
+
+            if (options && options.passwordPolicy) {
```

Comment:
What if I send a `{}` as `options` ? `defaultArgs` will not be applied and passwordPolicy will not be set.
Make sure you assign `passwordPolicy` into `options` if it's not set.